lib/govuk_navigation_helpers/content_item.rb

```diff
@@ -90,5 +99,32 @@ def hash
     def eql?(other)
       self == other
     end
+
+  private
+
+    def taxon_links
+      # A normal content item's taxon links are stored in ["links"]["taxons"]
+      # whereas a Taxon content item's taxon links are stored in ["links"]["parent_taxons"]
+      # so here we cater for both possibilities
+      Array(
+        content_store_response.dig("links", "taxons") ||
+          content_store_response.dig("links", "parent_taxons")
+      )
+    end
+
+    def descendent_from_whitelisted_root_taxon?(taxon)
+      root_taxon = get_parent_taxon(taxon)
+      ContentItem.whitelisted_root_taxon_content_ids.include?(root_taxon["content_id"])
+    end
+
+    def get_parent_taxon(taxon)
```

Comment:
Maybe get first level taxon would be a clearer name for this, as it's getting the taxon at that level of the taxonomy, rather than the immediate parent?